src/applications/appeals/10182/tests/10182-notice-of-disagreement.cypress.spec.js

```diff
@@ -101,6 +101,15 @@ const testConfig = createTestConfig(
         });
       },
 
+      'area-of-disagreement/:index': ({ afterHook /* , index */ }) => {
+        cy.injectAxeThenAxeCheck();
+        afterHook(() => {
+          cy.fillPage(); // temporary until page is updated with web components
+          // console.log('testing :index pageHooks', index);
```

Comment:
Do you still need that logger for anything?

Reply:
I added it there so that the platform team can test to make sure it works. I have another PR I'm about to open that'll completely replace this callback.